tests/e2e/integration/transactions.js

```diff
@@ -1,42 +1,20 @@
 import { TXS_PER_PAGE } from '../../../src/popup/utils/constants';
-import { testAccount } from '../../../src/popup/utils/config';
-import { formatDate, formatTime } from '../../../src/popup/utils';
 
 describe('Tests cases for transactions page', () => {
-  beforeEach(() => {
-    cy.login();
-  });
-
   it('Load transactions, load additional transactions on scroll', () => {
-    cy.openTransactions().get('[data-cy=list]').children().should('have.length', TXS_PER_PAGE);
+    cy.login()
+      .openTransactions()
+      .get('[data-cy=list]')
+      .children()
+      .its('length')
+      .should('be.gte', 0);
```

Comment:
isn't the length of anything is always greater than or equal to 0?
```suggestion
      .should('be.gt', 0);
```

Reply:
That is obviously a mistake, yes